### PR TITLE
updated/corrected German translations

### PR DESCRIPTION
**Pull request by stritti**
Signed-off-by: Stephan Strittmatter stephan.strittmatter@gmail.com

### CODE REVIEW

File: bundles/core/org.eclipse.smarthome.core.semantics/src/main/resources/tags_de.properties [hunk 1]
```diff
@@ -73,7 +73,7 @@ Equipment_Siren=Sirene,Sirenen
 Equipment_SmokeDetector=Rauchmelder
 Equipment_Speaker=Lautsprecher
 Equipment_Valve=Ventil,Ventile
-Equipment_WallSwitch=Wandschalter,Wanddimmer,Lichtschalter
+Equipment_WallSwitch=Wandschalter,Wanddimmer,Lichtschalter,Lichtschalter,Schalter
```
Comment (htreu) [inline bundles/core/org.eclipse.smarthome.core.semantics/src/main/resources/tags_de.properties]: `Lichtschalter` is now doubled

**Comment by htreu** [comment]
@stritti thanks for this contribution. Since we are an Eclipse project we need to have contributors sing the Eclipse Contributor Agreement (ECA). This means you have to create an account at eclipse.org and sign the ECA with the e-mail address provided in the signed-off-by footer. Please refer to the [Eclipse SmartHome contributor docs](https://www.eclipse.org/smarthome/documentation/community/contributing.html#conventions-for-pull-requests) for more details.

Thank you very much!

**Comment by stritti** [comment]
Oh, yes I see. I had previously commited to openHAb and thought it is  the same ...
It is done now.

**Comment by stritti** [comment]
Signed-off-by: Stephan Strittmatter stephan.strittmatter@gmail.com (github: stritti)

**Pull request merged**